docs/runtime_attributes.md

```diff
@@ -0,0 +1,22 @@
+# Key Runtime Attributes
+
+This doc is intended to capture the key runtime attributes for Node.js.  
+These attributes will be used along with the use-cases in order to
+guide what we develop/run as part of the community benchmarking
+efforts.
+
+# Attributes
+
+The following attributes have been identified so far:
+
+* startup/stop time
+* memmory footprint at startup
+* memory footprint after load
```

Comment:
Its a good point that "load" is not all that specific.  It will likely be different  depending on the bechmark but I'm not sure how to be more specific in this doc.   Maybe "memory footprint after load metrics are collected".  (timing will depend on the specific benchmark).   

For example, in acme air its after the transactions have been driven, or in the case of reading files as you mentioned after the load metrics are collected.